plugin-server/src/main/ingestion-queues/session-recording-v2/kafka/message-parser.ts

```diff
@@ -121,6 +122,12 @@ export class KafkaMessageParser {
         }
         const { validEvents, startDateTime, endDateTime } = result
 
+        const startDiff = Math.abs(startDateTime.diffNow('day').days)
+        const endDiff = Math.abs(endDateTime.diffNow('day').days)
+        if (startDiff >= MESSAGE_TIMESTAMP_DIFF_THRESHOLD_DAYS || endDiff >= MESSAGE_TIMESTAMP_DIFF_THRESHOLD_DAYS) {
+            return dropMessage('message_timestamp_diff_too_large')
```

Comment:
does drop message log an ingestion warning?

would be cool to have an ingestion warning but not blocking
(to be fair i don't know if users are really looking at ingestion warnings)

Reply:
https://github.com/PostHog/posthog/blob/master/plugin-server/src/main/ingestion-queues/session-recording-v2/kafka/message-parser.ts#L62

It prints a log warning and increments a stats counter